pcdsdevices/ipm.py

```diff
@@ -323,9 +329,9 @@ class IPIMB(BaseInterface, GroupDevice):
         Trigger component.
     """
 
-    tab_whitelist = ['isum', 'xpos', 'ypos']
+    tab_whitelist = ['sum', 'xpos', 'ypos']
 
-    isum = Cpt(EpicsSignalRO, ':SUM', kind='hinted')
+    sum = Cpt(EpicsSignalRO, ':SUM', kind='hinted')
```

Comment:
I don't even remember why this was isum

Reply:
Would recommend a `@property isum -> warn and return self.sum` for back-compat, unless you can say for certain there is no code using this.